libgeopmd/src/LevelZeroIOGroup.cpp

```diff
@@ -811,6 +811,10 @@ namespace geopm
             for (int domain_idx = 0; domain_idx < m_platform_topo.num_domain(
                                      signal_domain_type(sv.first)); ++domain_idx) {
                 try {
+                    if (sv.first.find("_PERFORMANCE_") != std::string::npos && !m_is_perf_factor_enabled) {
+                        unsupported_signal_names.push_back(sv.first);
+                        break;
+                    }
```

Comment:
What's this change doing here?  What does it fix re: this bug?

Reply:
It is unrelated.  @dannosliwcd sited the issue.  I can open a separate pull pull request it needed.

Reply:
I have updated the commit message just now.

Reply:
I didn't observe any problematic behavior, just found this through unplanned code inspection.

I was digging in this code while looking into adding memory bandwidth signal and found it odd that we check whether the signal's name matches name patterns of unsupported signals _after_ we attempt to read the signal (`sv.second.m_devpool_func(domain_idx);`).

So with the prior implementation, we try to read the signal, then we check whether it's a signal that we know we should avoid reading.